marklogic-data-hub/src/test/java/com/marklogic/hub/PiiE2E.java

```diff
@@ -0,0 +1,330 @@
+					e.printStackTrace();
+				}
+        	}      	
+        }); 
+        
+        // save pii, install user modules and deploy security 
+        EntityManager entityManager = EntityManager.create(getHubConfig());
+        entityManager.savePii();
+        
+        installUserModules(getHubConfig(), true);
+        deploySecurity();
+
+        //Clerk able to see harmonized document but not ssn
+        Assert.assertEquals("{\"fullName\":\"Ellie Holland\",\"worksFor\":\"SuperMemo Limited\",\"email\":\"ellie.holland@supermemolimited.biz\"}",getCustomerHistory(clerkClient, "Holland"));
+    	// Clerk unable to search by "ssn"
+    	Assert.assertEquals("{}",getCustomerHistoryBySSN(clerkClient, "228-80-9858"));
+    	
+    	//COmpliance oficer able to search using ssn and see all fields in harmonized document
+    	Assert.assertEquals("{\"fullName\":\"Ellie Holland\",\"worksFor\":\"SuperMemo Limited\",\"email\":\"ellie.holland@supermemolimited.biz\",\"ssn\":\"164-32-6412\"}",getCustomerHistory(officerClient, "Holland"));
+    	Assert.assertEquals("{\"fullName\":\"Melanie Douglas\",\"worksFor\":\"Erntogra Inc.\",\"email\":\"melanie.douglas@erntograinc.eu\",\"ssn\":\"228-80-9858\"}",getCustomerHistoryBySSN(officerClient, "228-80-9858"));   
+    }
+    
+    @Test
+    public void testSavePii() throws Exception {
+    	installEntities();
+    	EntityManager entityManager = EntityManager.create(getHubConfig());
+    	entityManager.savePii();
+    	
+        Path protectedPathConfig = getHubConfig().getUserSecurityDir().resolve("protected-paths");
+        verifyResults(protectedPathConfig);
+        
+        Path queryRolesetsConfig = getHubConfig().getUserSecurityDir().resolve("query-rolesets");
+        verifyResults(queryRolesetsConfig);
+    }
+    
+    private void verifyResults(Path path) throws IOException {
+    	JsonParser parser = new JsonParser(); 	
+    	Files.walk(path)
+    	.filter(f -> !Files.isDirectory(f))
+        .forEach(f ->{
+        	System.out.println(f.getFileName());
+           	URL url = PiiE2E.class.getClassLoader().getResource("pii-test/keys/"+f.toFile().getName());
+        	JsonObject expected = null;
+        	JsonObject actual = null;
+        	FileReader expectedReader = null;
+        	FileReader actualReader = null;
+			try {
+				actualReader = new FileReader(f.toFile());
+				expectedReader = new FileReader(Paths.get(url.toURI()).toFile());
+				actual = parser.parse(actualReader).getAsJsonObject();
+				expected = parser.parse(expectedReader).getAsJsonObject();
+			} catch (JsonIOException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			} catch (JsonSyntaxException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			} catch (FileNotFoundException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			} catch (URISyntaxException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();

```

Comment:
If you really want to just print stack traces for these, then go ahead and remove the TODO comments.  I think you might actually want to throw a RuntimeException like a "YourTestHanessIsScrewedUpException" :)